src/hotspot/share/runtime/abstract_vm_version.hpp

```diff
@@ -71,9 +71,10 @@ class Abstract_VM_Version: AllStatic {
   static int          _vm_build_number;
   static unsigned int _data_cache_line_flush_size;
 
+ public:
```

Comment:
Would protected work here instead of public?

Reply:
Hi David, unfortunately protected does not work here.